src/test/java/me/moodcat/backend/rooms/RoomBackendTest.java

```diff
@@ -167,30 +171,46 @@ public void playSongCreatesQueueWhenRepeating() {
         instance.playNext();
         instance.merge();
 
-        assertNotEquals(room.getPlayHistory(), room.getPlayQueue());
+        assertFalse(room.getPlayQueue().isEmpty());
```

Comment:
It is intentional, the purpose of the test is to verify that a queue is created when repeating, it broke because the history is now deleted when repeating. The purpose of the test hasn't changed.